config/routes.rb

```diff
@@ -3,5 +3,7 @@
   resources :events, only: [:show, :create, :new]
   resources :presentations, only: [:show, :create, :new]
 
+  get '/upcoming-events', to: 'events#upcoming'
```

Comment:
It might be nice to put this under the events resource:

```ruby
resources :events, only: :blah do
  get "/upcoming-events", on: :collection
end
```

(don't quote me on the exact implementation)